docs/StructureAndRationale.md

```diff
@@ -0,0 +1,75 @@
+# CTK/CTS Design - Structure and Rationale
+
+## Design Goals
+
+The intent of the CTK is to make it easy to run and write GA4GH API-centric tests, and to use the test results in various reporting environments.
+
+## Layout Overview
+
+The CTK is structured as a set of Maven modules:
+```
+                                                             +-----------------------------+
+ctk bash script, raw java invoke +--------------------------->ctk-cli                      |
```

Comment:
Should ctk-server be included somewhere in this chart? I had some difficulty understanding how that fit in until I grepped through the docs folder for 'ctk-server'